rustfmt.toml

```diff
@@ -38,6 +38,5 @@ ignore = [
 
     # these are ignored by a standard cargo fmt run
     "compiler/rustc_codegen_cranelift/y.rs", # running rustfmt breaks this file
```

Comment:
This has been replaced with a dedicated shell script a couple of days ago, but `y.rs` will be kept for a while for compatibility reasons.